azure-pipelines/1esmain.yml

```diff
@@ -18,6 +18,9 @@ parameters:
   - name: isOfficialBuild
     type: boolean
     default: true
+  - name: vsixFileName
+    type: string
+    default: ''

```

Comment:
Should the default name not be the name of the extension file?

Reply:
Yeah, by default it stays as `''` and isn't used.